src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf) format(ttf),
+    url(./fonts/Avenir-Heavy.ttf) format(ttf);
+}
```

Comment:
Fix according 7 point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and use different font-weight (from [mockup](https://www.figma.com/file/kf3AWulK9elrNk34wtpjPw/Airbnb-Search-bar?node-id=0%3A1) task requirement)